### DISCUSSION TITLE
"Remote file error" when trying to download with port forwarding?

### DISCUSSION THREAD

**Discussion by Merlin04** [nicotine-plus/nicotine-plus#2433] [Q&A]
I have a bit of a weird network setup - I'm using a Tailscale exit node with nginx running to expose the soulseek port on my laptop to the broader internet. When I'm disconnected from this, my port shows as closed in the tester linked in the settings panel of Nicotine+ and by the various chatbots that check for you, but I can download normally. However - when I'm connected to it the various testers show my port as open but I get a "Remote file error" most of the time when trying to download any file; somehow it sometimes works, but I have to retry the download a bunch before I can even get one file downloading. Is there any way to see more detailed logs about what's happening, or does anyone have any idea how to fix this?

**Comment by slook** [comment]
> way to see more detailed logs about what's happening

View > Show Log History Pane > Right click > Log Catagories > **"[Debug] Connections"**
